src/coreclr/jit/gentree.h

```diff
@@ -194,7 +194,7 @@ inline AssertionIndex GetAssertionIndex(unsigned index)
 
 class AssertionInfo
 {
-    // true if the assertion holds on the bbNext edge instead of the bbTarget edge (for GT_JTRUE nodes)
+    // true if the assertion holds on the false edge instead of the true edge (for GT_JTRUE nodes)
```

Comment:
We should consider renaming the field too, the logical name would be `m_isFalseEdgeAssertion` but that might mix up whether it's an assertion about the false edge, or a false assertion about an edge. So maybe `m_assertionHoldsOnFalseEdge`?

But I suggest deferring this until later.

Reply:
I have some cleanup tasks planned as a follow-up to this PR. I'll make sure to include the rename; `m_assertionHoldsOnFalseEdge` sounds good to me.